indico/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN ${INDICO_VIRTUALENV}/indico setup create_symlinks /opt/indico
 RUN ${INDICO_VIRTUALENV}/indico setup create_logging_config /opt/indico/etc
 
 COPY indico.conf /opt/indico/etc/indico.conf
-RUN ln -s /opt/indico/etc/indico.conf /etc/indico.conf
```

Comment:
are we setting INDICO_CONFIG somewhere? Otherwise removing this will probably result in indico not finding its config file.

Reply:
Yes, in 3rd line of this file. That's why I removed it becasue it;s redundant